### ISSUE TITLE
Access denied for public repository

### ISSUE THREAD

**Issue by mondeja** [lemurheavy/coveralls-public#1522]
More than 24 hours after change [Gihub repository](https://github.com/mondeja/waves) settings from private to public, I'm still getting Access Denied 403 page in coveralls.io.

- [Coverage submitted in action](https://github.com/mondeja/waves/runs/1814666450?check_suite_focus=true#step:7:12).
- [Link to job (Access denied)](https://coveralls.io/jobs/74437524) (even logged into my account)

I'm not sure if this is the best place to report it, or how long does it take to synchronize coveralls with repository access permissions. I've tried also pushing all the buttons of synchronization in coveralls dashboard, without results.

**Comment by afinetooth** [comment]
Hi @mondeja, there is a setting called REPO INFO CHANGE > SYNC in REPO SETTINGS used to update the privacy setting for your repo. 

You'll want to use it whenever key info has been updated at Github, including the privacy setting:

![Screen Shot 2021-02-02 at 6 31 53 PM](https://user-images.githubusercontent.com/918667/106689497-ee1e8c80-6584-11eb-8d2a-cf2f0130c3d8.png)

Here are the settings for your project:
https://coveralls.io/github/mondeja/waves/settings

---

I synced for you and the privacy icon (lock) immediately disappeared from your repo. 

Let me know if that doesn't resolve your issue.
